src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -140,15 +152,107 @@ public LibraryTab(JabRefFrame frame,
         });
     }
 
+    public void setDataLoadingTask(BackgroundTask<ParserResult> dataLoadingTask) {
+        this.dataLoadingTask = dataLoadingTask;
+    }
+
+    public BackgroundTask<?> getDataLoadingTask() {
+        if (dataLoadingTask == null) {
```

Comment:
This condition/if is uncessary as you either return null or the dataLoadingTask.